scripts/api_dump_generator.py

```diff
@@ -176,11 +176,45 @@
     return result;
 }}
 
+VkResult util_GetExtensionProperties(const uint32_t count, const VkExtensionProperties *layer_extensions, uint32_t *pCount,
```

Comment:
util_GetExtensionProperties is also defined inside of screenshot or monitor (dont remember which). Rather than being hardcoded in api_dump, it should live in vk_layer_table.h so that all 3 layers can use the same implementation.

Reply:
👍🏾